# Request
geometry_msgs/Pose pose
---
# Response
bool status